packages/react-resizable-panels/src/Panel.ts

```diff
@@ -90,7 +93,7 @@ export function PanelWithForwardedRef({
   ...rest
 }: PanelProps & {
   forwardedRef: ForwardedRef<ImperativePanelHandle>;
-}) {
+}): ReactNode {
```

Comment:
When I added `keyof HTMLElementTagNameMap`, declaration generation failed despite `tsc` being happy. The error message was a red herring - the components just need an explicit return type. Not sure what to make of that.

Maybe just `ReactElement` is more appropriate?

Reply:
That's interesting. Not sure why that is, but I think `ReactNode` makes sense as the return type 👍🏼